### PR TITLE
Added rulesDirectory option

### PR DESCRIPTION
**Pull request by buzinas**

### CODE REVIEW

File: lib/init.coffee [hunk 1]
```diff
@@ -3,9 +3,18 @@ path = require 'path'
...
+  config:
+    rulesDirectory:
+      type: 'string'
+      title: 'Custom rules directory'
+      default: ''
+
   activate: ->
     @subscriptions = new CompositeDisposable
     @scopes = ['source.ts', 'source.tsx']
+    @subscriptions.add atom.config.observe 'linter-php.rulesDirectory',
```
Comment (Arcanemagus) [inline lib/init.coffee]: You'll need to change that to `'linter-tslint.rulesDirectory'`.

File: package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "name": "linter-tslint",
   "main": "./lib/init",
   "linter-package": true,
-  "version": "0.4.3",
+  "version": "0.5.3",
```
Comment (Arcanemagus) [inline package.json]: Don't update the `package.json` here, `apm publish` will do that automatically.

**Comment by buzinas** [comment]
@Arcanemagus fixed the mentioned issues :)

**Comment by Arcanemagus** [comment]
Looks like this should work, have you tested locally?

**Comment by Arcanemagus** [comment]
Also if you put "Fixes #37" in the description that issue will automatically be closed when this is merged.

**Comment by buzinas** [comment]
@Arcanemagus found a way to test locally and it's not working as expected.

It uses the `.atom/packages` as home directory, and I'm trying to get the projects' directory, so I can finish this and commit.

**Comment by buzinas** [comment]
OK, @Arcanemagus, now it's working as expected :)

File: lib/init.coffee [hunk 3]
```diff
@@ -20,9 +31,16 @@ module.exports =
         filePath = textEditor.getPath()
         text = textEditor.getText()
         configuration = Linter.findConfiguration(undefined, filePath)
+        
+        if (rulesDirectory && textEditor.project && textEditor.project.getPaths().length)
+          directory = textEditor.project.getPaths()[0] + '/' + rulesDirectory
```
Comment (Arcanemagus) [inline lib/init.coffee]: Use path.sep instead of `'/'`.

Comment (buzinas) [inline lib/init.coffee]: Done :)

**Comment by buzinas** [comment]
@Arcanemagus  Ready to merge, I guess!

**Comment by buzinas** [comment]
@Arcanemagus thanks very much for the help :)

**Comment by Arcanemagus** [comment]
Published as v0.5.0.

Thanks for taking the time to work on this :+1:.

**Pull request merged**